eng/versioning/version_client.txt

```diff
@@ -25,6 +25,7 @@ com.azure:azure-sdk-template;1.0.4-beta.2;1.0.4-beta.2
 com.azure:azure-storage-blob;12.2.0-beta.1;12.2.0-beta.2
 com.azure:azure-storage-blob-batch;12.1.0;12.2.0-beta.1
 com.azure:azure-storage-blob-cryptography;12.1.0;12.2.0-beta.1
+com.azure:azure-storage-blob-nio;12.0.0-preview1;12.0.0-preview2
```

Comment:
Should use `12.0.0-beta.1` and maybe `beta.2`, though I think both should be the same given that this is the first preview for this package.